gui/src/components/LinkRenderer.svelte

```diff
@@ -3,4 +3,4 @@
   export let title = undefined;
 </script>
 
-<a {href} {title} target="_blank"><slot /></a>
+<a {href} {title} target="_blank" rel="noreferrer"><slot /></a>
```

Comment:
What was the reasoning behind using `noreferrer` here and other places?

Reply:
VSCode was giving me a security warning saying anything with `target="_blank"` should have `rel="noreferrer"`, maybe not a concern for our use-case here, but I figured it didn't hurt.